<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userStatistics</class>
 <widget class="qMRMLWidget" name="userStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>571</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Statistics table node:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="tableNodeSelector">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLTableNode</string>
        </stringlist>
       </property>
       <property name="baseName">
        <string>UserStatisticsTableNode</string>
       </property>
       <property name="noneEnabled">
        <bool>false</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="qMRMLTableView" name="statisticsTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Merge statistics</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Statistic table nodes:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLCheckableNodeComboBox" name="mergeTablesNodeSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLTableNode</string>
         </stringlist>
        </property>
        <property name="selectNodeUponCreation">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="mergeTablesButton">
        <property name="text">
         <string>Merge</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLTableView</class>
   <extends>QTableView</extends>
   <header>qMRMLTableView.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>tableNodeSelector</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>statisticsTable</receiver>
   <slot>setMRMLTableNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userStatistics</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>tableNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userStatistics</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>statisticsTable</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userStatistics</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>mergeTablesNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
